lib/mongo/operation/write/command/update.rb

```diff
@@ -45,11 +46,14 @@ class Update
           #
           # @return [ Hash ] The selector describing this update operation.
           def selector
-            { update: coll_name,
-              updates: updates,
-              writeConcern: write_concern.options,
-              ordered: ordered?
+            cmd = {
+                update: coll_name,
+                updates: updates,
+                writeConcern: write_concern.options,
+                ordered: ordered?
             }
+            cmd['bypassDocumentValidation'] = true if bypass_document_validation
+            return cmd
```

Comment:
Firstly, the return value of a Ruby method is the last expression evaluated, so you would just write

```ruby
cmd
```

as the last line.
But to answer your question, you could do

```ruby
bypass_document_validation ? cmd.merge!('bypassDocumentValidation' => true) : cmd
```